content-resources/src/main/java/fi/nls/oskari/db/BundleHelper.java

```diff
@@ -89,4 +90,15 @@ public static Bundle getRegisteredBundle(final String id, Connection conn) throw
             }
         }
     }
+
+    public static void removeBundle(final Bundle bundle, Connection conn) throws SQLException {
```

Comment:
You should just have String bundleName instead of Bundle as param so it's easier to see what this method needs/uses (it could getId() from bundle). Also you should rename this to unregisterBundle() as it's an opposite of registerBundle()

Reply:
Updated method name and parameters as proposed